### PR TITLE
Replace Gesso hero components with USWDS hero

### PR DESCRIPTION
**Pull request by kmonahan**

### CODE REVIEW

**Review by dcmouyard — CHANGES_REQUESTED** [review]
Looks great!

Since we got rid of the old hero components, we need to update the hero image in `_patterns/02-base/images/hero-image.twig` to match the dimensions that the USWDS hero is expecting. The data for the hero image details are in `_data/data.yml`.

**Comment by kmonahan** [comment]
Updated!

File: source/_data/data.yml [hunk 1]
```diff
@@ -22,10 +22,10 @@ img:
     height: '105'
     width: '140'
   hero:
-    src: 'http://fpoimg.com/1600x800?text=Hero 2:1'
-    alt: 'Hero 2:1'
-    height: '800'
-    width: '1600'
+    src: 'http://fpoimg.com/1600x800?text=Hero 5:2'
```
Comment (dcmouyard) [inline source/_data/data.yml]: This image src should also be 2048x820.

**Review by dcmouyard — CHANGES_REQUESTED** [review]

File: source/_data/data.yml (hunk 1 above)
Comment (kmonahan) [reply source/_data/data.yml]: So it should. Just fixed!

**Review by dcmouyard — APPROVED** [review]
👍

**Pull request merged**